extensions/amp-story/1.0/amp-story.css

```diff
@@ -984,3 +984,8 @@ amp-story-page .i-amphtml-story-page-open-attachment-label {
   text-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36) !important;
   white-space: nowrap !important;
 }
+
+amp-story-quiz {
+  width: 100% !important;
```

Comment:
How does this look on a tablet like an iPad, or on landscape orientation (with `supports-landscape`)?
Can you still tap anywhere in between the quiz and the safety area to navigate?

Reply:
In addressing the review comments, I added a max-width of 400px, conforming to the UX spec, and I centered the content by default (though it is overridable).

With the constrained width and centering it looks much better on an iPad or other tablet.

As far as the tap safety area, it works the same in mobile landscape mode - though iPads in landscape mode fallback to the desktop story viewer because of the width. The desktop story viewer removes the safety area entirely (likely due to distance from the actual edge of the screen and not the virtual one). This is not a problem with tapping on the sides, though it will be a problem with CTAs and Page Attachments, as the quizzes remain interactive within their buffer area.

Reply:
I think this behavior is ok for quizzes in desktop mode stories, but what I have done is opened issue #25871 to increase the size of the tap targets for page attachments